### PR TITLE
Genome build check on case load

### LINKED ISSUE #2300: Introduce genome build check when loading a new case
See comment in #2298.
Basically check that the config provide either genome build 37 or 38, or that a genome build param is passed whenever cases are uploaded using ped files, because in current master there is no way to pass genome build when these cases are created

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2300 .

This PR makes sure that any case loaded into Scout (via .yaml, .py, or .ped files) will have a genome build set in the database. Accepted values will only be 37 or 38.

**How to test**:
1. Using demo data and a demo instance of Scout, try to load (reupload)  a case using the demo yaml file or the ped file.
2. When you use the ped file it will ask you to provide a genome build for the case that will be saved in the database later.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Codefactor is complaining for some code located in another test down in the file. Looks complicated to fix so I'll just leave it that way

**Comment by dnil** [comment]
This looks nice and ready! Have you tried on stage?

**Comment by northwestwitch** [comment]
> This looks nice and ready! Have you tried on stage?

Not yet. I'll do it

File: scout/commands/load/case.py [hunk 1]
```diff
@@ -106,6 +106,13 @@ def case(
         LOG.debug("Stack trace: {}".format(traceback.format_exc()))
         raise click.Abort()
 
+    if config_data.get("human_genome_build") not in [37, 38]:
```
Comment (dnil) [inline scout/commands/load/case.py:109]: 👍

**Review by dnil — APPROVED** [review]
This looks well tested; as long as this is not missing from cg generated configs it should be fine.

**Comment by northwestwitch** [comment]
> This looks well tested; as long as this is not missing from cg generated configs it should be fine.

So it's fine to merge without testing on stage? 😄

File: scout/commands/load/case.py (hunk 1 above)
Comment (northwestwitch) [reply scout/commands/load/case.py:109]: And that is the point, it shouldn't be missing or different from 37 and 38 in the config files

**Comment by dnil** [comment]
Well, you will anyway want to log on and check that one of the live cases (e.g. keepviper) has a relevant build line generated by cg (which I think it does) so perhaps just install it and hit scout load case when you are at it? 😊

**Comment by northwestwitch** [comment]
> so perhaps just install it and hit scout load case when you are at it

Ok! Tomorrow!

**Comment by northwestwitch** [comment]
Checking the config files produced by cg: they contain '37' instead of 37:

image

I'll modify the code to take this into account! Good that you made me test it @dnil 🙏

**Comment by northwestwitch** [comment]
Branch installed on Hasta stage:

image
blablabla
image

 ----
Upload keenviper via scout-stage: 🆗 

image

image

image

 ----

Upload keenviper via cg-stage:

image

image


The upload just hangs!

**Comment by northwestwitch** [comment]
Ah no, cg-stage works:

image

It's just not printing anything for a long time and I loose hope easily! 😆

**Comment by northwestwitch** [comment]
Merging in 3,2, ..

**Pull request merged**